openstack/baremetal/v1/nodes/requests.go

```diff
@@ -181,6 +181,10 @@ type CreateOptsBuilder interface {
 
 // CreateOpts specifies node creation parameters.
 type CreateOpts struct {
+
+	// The interface to configure automated cleaning for a Node
+	AutomatedClean bool `json:"automated_clean,omitempty"`
```

Comment:
This will default to `false` I assume? You need to allow explicit `true` and `false`, skipping completely if no value is provided (so `*bool` I guess).

Reply:
`*bool` and `omitempty` should do it. And please also add which microversion this was added as part of the docstring/comment.

https://github.com/gophercloud/gophercloud/blob/master/docs/MICROVERSIONS.md#new-response-fields

Reply:
Fixed, thanks.